test/bit_test.c

```diff
@@ -3,7 +3,9 @@
 #ifdef __APPLE__
```

Comment:
This part should be moved to zeda_compat.h.
(I don't remember if I wrote this...)